DEPERECATED.md

```diff
@@ -0,0 +1,3 @@
+# DEPRECATED starting from 1.3.0
```

Comment:
Update the file name DEPERECATED.md -> DEPRECATED.md now, otherwise we will have to deprecate the DEPERECATED.md at a later date!